src/styles/_colors.scss

```diff
@@ -0,0 +1,8 @@
+$primaryBlack: #000;
+$secondaryBlack: #333;
+$primaryWhite: #fff;
+$secondaryWhite: #f2f2f2;
+$primaryGray: gray;
+$secondaryGray: lightgray;
+$tartiaryGray: #ccc;
+$buttonPrimary: #007bff;
```

Comment:
Using three-digit hex codes for colors is not considered a best practice. It's more common to use six-digit hex codes for colors in order to provide more precision and flexibility. Also use hex code instead of color name.